test/unit/alignment/matrix/detail/two_dimensional_matrix_test.cpp

```diff
@@ -78,249 +172,183 @@ TYPED_TEST(two_dimensional_matrix_test, rows)
+    this->expect_eq(this->matrix.at({row_index_type{0u}, column_index_type{1u}}), this->expected_matrix_content[1]);
+    this->expect_eq(this->matrix.at({row_index_type{0u}, column_index_type{2u}}), this->expected_matrix_content[2]);
+    this->expect_eq(this->matrix.at({row_index_type{0u}, column_index_type{3u}}), this->expected_matrix_content[3]);
+    this->expect_eq(this->matrix.at({row_index_type{1u}, column_index_type{0u}}), this->expected_matrix_content[4]);
+    this->expect_eq(this->matrix.at({row_index_type{1u}, column_index_type{1u}}), this->expected_matrix_content[5]);
+    this->expect_eq(this->matrix.at({row_index_type{1u}, column_index_type{2u}}), this->expected_matrix_content[6]);
+    this->expect_eq(this->matrix.at({row_index_type{1u}, column_index_type{3u}}), this->expected_matrix_content[7]);
+    this->expect_eq(this->matrix.at({row_index_type{2u}, column_index_type{0u}}), this->expected_matrix_content[8]);
+    this->expect_eq(this->matrix.at({row_index_type{2u}, column_index_type{1u}}), this->expected_matrix_content[9]);
+    this->expect_eq(this->matrix.at({row_index_type{2u}, column_index_type{2u}}), this->expected_matrix_content[10]);
+    this->expect_eq(this->matrix.at({row_index_type{2u}, column_index_type{3u}}), this->expected_matrix_content[11]);
+
+    EXPECT_THROW((this->matrix.at({row_index_type{3u}, column_index_type{3u}})), std::invalid_argument);
+    EXPECT_THROW((this->matrix.at({row_index_type{2u}, column_index_type{4u}})), std::invalid_argument);
 }
 
-TYPED_TEST(two_dimensional_matrix_test, conversion)
+TYPED_TEST(two_dimensional_matrix_test, construction_other_order)
 {
-    using matrix_type = typename TestFixture::matrix_type;
-
-    matrix_type matrix{number_rows{3}, number_cols{4}};
-    std::ranges::copy(this->expected_range, matrix.begin());
-
-    if constexpr (TypeParam::value == matrix_major_order::row)
-    {
-        using converted_t = two_dimensional_matrix<uint32_t, std::allocator<uint32_t>, matrix_major_order::column>;
-        converted_t converted = static_cast<converted_t>(matrix);
-        EXPECT_EQ(converted.rows(), matrix.rows());
-        EXPECT_EQ(converted.cols(), matrix.cols());
-        EXPECT_EQ((converted[{row_index_type{0u}, column_index_type{0u}}]), 0u);
-        EXPECT_EQ((converted[{row_index_type{1u}, column_index_type{0u}}]), 4u);
-        EXPECT_EQ((converted[{row_index_type{0u}, column_index_type{1u}}]), 1u);
-        EXPECT_EQ((converted[{row_index_type{0u}, column_index_type{3u}}]), 3u);
-        EXPECT_EQ((converted[{row_index_type{2u}, column_index_type{0u}}]), 8u);
-        EXPECT_EQ((converted[{row_index_type{2u}, column_index_type{3u}}]), 11u);
-    }
-    else
+    // Test that changing the matrix layout works.
+    static constexpr matrix_major_order other_matrix_order = TestFixture::matrix_order == matrix_major_order::row ?
+                                                             matrix_major_order::column : matrix_major_order::row;
```

Comment:
can you please wrap the command like so:
```
cond 
    ? on_true
    : on_false
```

Reply:
I couldn't find that rule in https://github.com/seqan/seqan3/wiki/General-formatting, but I will change it for you :heart: